tests/ShadowDOM/js/wrappers.js

```diff
@@ -94,6 +94,23 @@ suite('Wrapper creation', function() {
     assert.isTrue(Object.getPrototypeOf(br).hasOwnProperty('clear'));
   });
 
+  // Super class is defined as a class above the existing wrapped class (GeneratedWrapper).
+  test('Super class methods are wrapped as methods on base class', function() {
+      // Set a method on a prototype whose parent prototype is not yet wrapped
```

Comment:
also noticed this is +4 indent instead of +2 like the other tests
(fwiw, I'm personally more a fan of +4, but existing style of the code was +2 ... I guess due to Google JS style guide)